packages/fxa-settings/src/pages/Signup/container.tsx

```diff
@@ -81,8 +81,13 @@ const SignupContainer = ({
 
   const beginSignupHandler: BeginSignupHandler = useCallback(
     async (email, password, options) => {
-      options.verificationMethod = 'email-otp';
-      options.keys = isOAuthIntegration(integration);
+      const service = integration.getService();
```

Comment:
It appears we always override the service? Yet there are places we send it in as an option. Do we want to allow 'service' to be an option? If so should we respect the option provided?

(IMO, we should just rely on serivce here if possible and not consider it an option, ie Remove service from the BeginSignUpOptions type.)

Reply:
Updated. The service is no longer received as an option, and is only retrieved in the container component to be passed on to the endpoint.